plotter/labels.go

```diff
@@ -80,8 +79,7 @@ func (l *Labels) Plot(c draw.Canvas, p *plot.Plot) {
 		if !c.Contains(pt) {
 			continue
 		}
-		pt.X += l.XOffset
-		pt.Y += l.YOffset
+		pt = pt.Add(l.Offset)
```

Comment:
not sure whether we shouldn't move the offset fixup *before* the `c.Contains(pt)` condition...